src/App.js

```diff
@@ -10,10 +11,36 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    savedTabs: [...tabs],
+    tab: '',
+    selectedTab: tabs[0],
+  }
```

Comment:
just save selectedTab

```suggestion
  state = {
    selectedTab: tabs[0],
  }
```